### ISSUE TITLE
fastcancel option

### ISSUE THREAD

**Issue by ChrisRedacted** [fastbuild/fastbuild#497]
Is this still an experimental feature? What kinds of issues should we watch out for if we use it?

**Comment by ffulin** [comment]
-fastcancel is still considered experimental in v0.96, though I have been using it in production since soon after it the feature was added.

As of 10 minutes ago, I fixed a rare Windows-only with distributed compilation bug. Bug: (https://github.com/fastbuild/fastbuild/issues/486). Fix:(https://github.com/fastbuild/fastbuild/commit/2fccad78ef5f7e5c59633c8d3b3e4acff275de90).

This was the only known problem, and it was very rare (at least for me), and not harmful (after a failure, the next build would pass).

Assuming no new issues arise, in v0.97 this feature will no longer be considered experimental. It will eventually be an always on feature.

It should be noted that as of v0.96 Linux/OSX do not implement full process termination, and as such a) don't suffer from the above bug, but also b) don't necessarily terminate the process tree correctly. (Despite being heavily used in production environments, OSX and Linux are still considered to be in Alpha: they are not at feature parity with Windows)

I hope that helps. Let me know if you want more details.
